src/main/java/lotto/model/Rank.java

```diff
@@ -0,0 +1,44 @@
+package lotto.model;
+
+import java.util.Arrays;
+
+public enum Rank{
+    FIRST(6,2000000000),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
```
언더스코어 (_) 를 활용하여 가독성을 살릴 수 있습니다 :)